integrations/vault/secrets/database/src/main/java/module-info.java

```diff
@@ -20,6 +20,7 @@
 module io.helidon.integrations.vault.secrets.database {
     requires java.logging;
 
+    requires io.helidon.common.reactive;
```

Comment:
This module exposes an API `DbSecretsRx` that uses `Single`, we should use `requires transitive`. CC @tomas-langer

Reply:
Fixed